framework/auth/core.py

```diff
@@ -164,10 +165,18 @@ def logged_in(self):
     def from_kwargs(cls, request_args, kwargs):
         user = request_args.get('user') or kwargs.get('user') or _get_current_user()
         private_key = request_args.get('view_only')
```

Comment:
What's the effect of querying for `None` (our new behavior) vs. `''` (our old behavior)?